code/modules/reqs/supply.dm

```diff
@@ -282,7 +282,7 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 	. = ..()
 	if(.)
 		return
-	if(!allowed(user))
+	if(isliving(user) && !allowed(user))
 		return
```

Comment:
is this supposed to be a negation?

Reply:
This is correct. The function should not fail when the user is dead. allowed() works on id cards so only needs to be called if the user is living.

I could write this but it seems unnecessary with the nesting:

if(isliving(user))
	if(!allowed(user))
		return